### ISSUE TITLE
Adjust wording in "Importing data into Open Refine"

### ISSUE THREAD

**Issue by riannz** [LibraryCarpentry/lc-open-refine#208]
In the lesson titled 'Importing data into Open Refine':
Create your first OpenRefine project (using provided data)

Point 9 says the project name box is in the upper right corner, but in fact it is located in the middle of the blue menu bar above the columns of data. 

Going Further:
"Look at the other options on the Import screen - try changing some of these options and see how that changes the Preview and how the data appears after import."
Being very new to all this I could not see how to try some of the other options on the Import screen, as by the time you read this sentence you have already clicked 'Create Project' and the Import screen/preview window has gone.  It took me awhile to figure out that I would need to click on the diamond icon in the top left corner, to start over and create a new project in order to experiment with some of the other options on the Import screen.

**Comment by pab491** [comment]
I agree that shifting the order to allow for some practice with the variety of options should be included in the exercise before  'Create Project' is clicked. To add to this question of wording/ order of instructional points, I think it would also be beneficial to shift the 'What Should I know When Working With Open Refine' section from the introduction episode to the  importing data episode. To me, it seems that items under this section such as how, where, and how often the project data is saved would be more easily demonstrated and answer questions about what is actually happening to a dataset once it is imported to Open Refine.

(NYU/ Check Out)

**Comment by jas58** [comment]
Thank you for your comments to improve the Open Refine lesson.  Would either of you have time to review the current version of either lesson and then follow the steps in this "[Open a PR in Github tutorial](https://youtu.be/BpEWDg8AbPc)" to let us know where you see this edit and what words you would suggest?

**Comment by jas58** [comment]
@ErinBecker An additional point for instructor notes sprint I hope to get back to.
